.github/pull_request_template.md

```diff
@@ -0,0 +1,55 @@
+<!-- Thank you for contributing! -->
+
+## What is the purpose of this pull request? <!-- (put an "X" next to an item) -->
+
+- [ ] New Icon
+- [ ] Bug fix
+- [ ] New Feature
+- [ ] Documentation update
+- [ ] Other
+
+### Description
+<!-- Please insert your description here and provide info about the "what" this PR is contribution -->
+
+### Icon use case <!-- ONLY for new icons, remove this part if not icon PR -->
+<!-- What is the purpose of this icon? Please insert a real life use case of icons, text like "it's a "car" icon is not accepted." -->
+
+### Alternative icon designs <!-- ONLY for new icons, remove this part if not icon PR -->
+<!-- If you have any alternative icon designs, please attach them here. -->
+
+## Icon Design Checklist <!-- ONLY for new icons, remove this part if not icon PR -->
+
+#### Concept <!-- ONLY for new icons -->
+
+- [ ] Icons has a valid use case.
+- [ ] Icons do not represent a brand or logo.
+- [ ] Icons do not contain hate symbols.
+- [ ] Icons do not include religious or political imagery.
+- [ ] Icons is at most based on other Lucide icons and not on other people's work
+- [ ] Icons should not be a rotated or mirrored version of an already existing icon
+
+#### Naming <!-- ONLY for new icons -->
+
+- [ ] Icons are named in lowercase and use dashes instead of spaces, for example, `arrow-up` instead of `Arrow Up` and do not end with a number like -2, -3, -4, etc.
+- [ ] Icons are named by what they are rather than their use case, for example save should rather be named to floppy-disk
+- [ ] Icons that are part of a group are named group-type so, for example, badge-plus is based on badge
+- [ ] Provided a meta JSON file in `icons/[iconName].json`
```

Comment:
```suggestion
- [ ] Icons are named by what they are rather than their use case, for example save should rather be named to floppy-disk.
- [ ] Icons that are part of a group are named group-type so, for example, badge-plus is based on badge.
- [ ] Provided a meta JSON file in `icons/[iconName].json`.
```